pkg/api/v1alpha1/conversion.go

```diff
@@ -139,6 +139,14 @@ func V1alpha1ToInternal(
 		}
 	}
 
+	evictDaemonSetPods := false
```

Comment:
`evictDaemonSetPods` needs to be wired to `DefaultEvictorArgs`

Reply:
Would you also create a unit test for the conversion? To make sure the conversion produces the expected outcome.

Reply:
Hi @ingvagabund, thank you for the review.

Added the missing reference in the last commit for `DefaultEvictorArgs`.

About the unit test for conversion, where should be the ideal place to write it? I'm little lost if it should be within `strategymigration_test.go` or other file that I may be missing.

Reply:
https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/policyconfig_test.go#L43 under `TestV1alpha1ToV1alpha2`

Reply:
Added the missing tests